regression/storage.js

```diff
@@ -61,200 +61,200 @@ function setHash(obj, file, done) {
     });
   });
 
-  after(function(done) {
-    deleteFiles(bucket, function(err) {
+  after(function (done) {
+    deleteFiles(bucket, function (err) {
       assert.ifError(err);
       bucket.delete(done);
     });
   });
 
-  describe('acls', function() {
+  describe('acls', function () {
     var USER_ACCOUNT = 'user-spsawchuk@gmail.com';
 
-    describe('buckets', function() {
-      it('should get access controls', function(done) {
-        bucket.acl.get(done, function(err, accessControls) {
+    describe('buckets', function () {
+      it('should get access controls', function (done) {
+        bucket.acl.get(done, function (err, accessControls) {
           assert.ifError(err);
           assert(Array.isArray(accessControls));
           done();
         });
       });
 
-      it('should add entity to default access controls', function(done) {
+      it('should add entity to default access controls', function (done) {
         bucket.acl.default.add({
           scope: USER_ACCOUNT,
           role: storage.acl.OWNER_ROLE
-        }, function(err, accessControl) {
+        }, function (err, accessControl) {
           assert.ifError(err);
           assert.equal(accessControl.role, storage.acl.OWNER_ROLE);
 
           bucket.acl.default.get({
             scope: USER_ACCOUNT
-          }, function(err, accessControl) {
+          }, function (err, accessControl) {
             assert.ifError(err);
             assert.equal(accessControl.role, storage.acl.OWNER_ROLE);
 
             bucket.acl.default.update({
               scope: USER_ACCOUNT,
               role: storage.acl.READER_ROLE
-            }, function(err, accessControl) {
+            }, function (err, accessControl) {
               assert.ifError(err);
               assert.equal(accessControl.role, storage.acl.READER_ROLE);
 
-              bucket.acl.default.delete({ scope: USER_ACCOUNT }, done);
+              bucket.acl.default.delete({scope: USER_ACCOUNT}, done);
             });
           });
         });
       });
 
-      it('should get default access controls', function(done) {
-        bucket.acl.default.get(function(err, accessControls) {
+      it('should get default access controls', function (done) {
+        bucket.acl.default.get(function (err, accessControls) {
           assert.ifError(err);
           assert(Array.isArray(accessControls));
           done();
         });
       });
 
-      it('should grant an account access', function(done) {
+      it('should grant an account access', function (done) {
         bucket.acl.add({
           scope: USER_ACCOUNT,
           role: storage.acl.OWNER_ROLE
-        }, function(err, accessControl) {
+        }, function (err, accessControl) {
           assert.ifError(err);
           assert.equal(accessControl.role, storage.acl.OWNER_ROLE);
 
-          bucket.acl.get({ scope: USER_ACCOUNT }, function(err, accessControl) {
+          bucket.acl.get({scope: USER_ACCOUNT}, function (err, accessControl) {
```

Comment:
Similarly, the spaces between the `{ }` helps for readability. Please revert these spacing changes.